src/h2_connection.erl

```diff
@@ -773,7 +776,10 @@ route_frame(
     lager:debug("[~p] Received WINDOW_UPDATE Frame for Stream ~p",
                 [Conn#connection.type, StreamId]),
 
+
     Stream = h2_stream_set:get(StreamId, Streams),
+
+    lager:error("Before : ~p", [h2_stream_set:recv_window_size(Stream)]),
```

Comment:
** looks around **

** force pushes **

I have no idea what you're talking about :P